oak_runtime/Cargo.toml

```diff
@@ -53,7 +53,10 @@ serde_json = "*"
 sha2 = "^0.9.1"
 tink-core = { version = "^0.1", features = ["json"] }
 tink-aead = "^0.1"
+tink-awskms = "^0.1"
```

Comment:
Is it feasible to make KMS support feature-gated? These KMS support dependencies worry me, and I would prefer to not always include it in the TCB unless someone really wants to use KMS integration.

Reply:
Good point, will do.